README.md

```diff
@@ -1,33 +1,25 @@
-[Mapbox GL is an ecosystem of libraries for rendering maps using WebGL/OpenGL on the web, iOS, Android, Qt, macOS and more](https://www.mapbox.com/maps/). This repository holds the files and issues shared among all platforms.
+# Mapbox GL JS
 
-[<img width="981" alt="Mapbox GL gallery" src="https://cloud.githubusercontent.com/assets/281306/14547142/a3c98294-025f-11e6-92f4-d6b0f50c8e89.png">](https://www.mapbox.com/gallery/)
-
-## Mapbox GL JS
+Mapbox GL JS is a JavaScript library that renders interactive maps from vector tiles and Mapbox styles using WebGL. Mapbox GL JS is part of the [cross-platform Mapbox GL ecosystem](https://www.mapbox.com/maps/), which also includes compatible native SDKs for applications on [Android](https://www.mapbox.com/android-sdk/), [iOS](https://www.mapbox.com/ios-sdk/), [macOS](https://github.com/mapbox/mapbox-gl-native/tree/master/platform/macos), and [Qt](https://github.com/mapbox/mapbox-gl-native/tree/master/platform/qt).
```

Comment:
Fixed in https://github.com/mapbox/mapbox-gl-js/commit/6caa263658877c4d4f62d42bcb54d171bf8c87e5. Thanks!